README.md

```diff
@@ -142,3 +142,30 @@ an input and returns a str. By default, `RichHelpFormatter` sets the function to
 ```python
 RichHelpFormatter.group_name_formatter = str.title
 ```
+
+## Contributing
+Start by [forking the repo](https://github.com/hamdanal/rich-argparse/fork) and cloning your fork.
+
+For those using [VS Code](https://code.visualstudio.com/) there's [preconfigured settings](.vscode/) in the repo.
```

Comment:
There is no need to link to VS Code. If some one uses it they already know what it is.

Reply:
is there a reason _not_ to link to it? i only discovered VS Code recently, and i think it was through some github readme

Reply:
It is just not relevant for a reader of this document. I see it a small inconvenience/distraction that we are better off.